src/components/BirthdayMessage/index.jsx

```diff
@@ -33,7 +33,7 @@ const BirthdayMessage = ({ open, setOpen, name }) => {
 
   const Banner = width >= 1200 ? BannerLarge : BannerSmall;
 
-  return isAuthenticated && isBirthday ? (
+  return isAuthenticated && !isBirthday ? (
```

Comment:
Please don't merge this change to the repository. I'm guessing you were just testing this feature.

Reply:
Thank you for all the feedback. I made a mistake of not creating a separate branch specifically for this project, so a lot of the changes in this PR were unrelated to the project. I will fix them right now.

Reply:
Never anything wrong with opening a PR for review! I'm just a little bit feisty on reviewing when I see a new PR :)

Reply:
Never anything wrong with opening a PR for review! I'm just a little bit feisty on reviewing when I see a new PR :)